src/ImGui.NET/ImGui.cs

```diff
@@ -140,9 +140,16 @@ public static void Image(IntPtr userTextureID, Vector2 size, Vector2 uv0, Vector
             return ImGuiNative.igImageButton(userTextureID, size, uv0, uv1, framePadding, backgroundColor, tintColor);
         }
 
+        //obsolete!
         public static bool CollapsingHeader(string label, string id, bool displayFrame, bool defaultOpen)
+        {   
+            int default_open_flags = 1 << 5;
```

Comment:
That's probably a good idea. Do you want to maintain separate file per struct/enum, or should I put it into some existing one?